src/components/Contacts/ContactDetails/ContactTaskRow/ContactTaskRow.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, styled, Typography } from '@material-ui/core';
+import { TFunction } from 'i18next';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ActivityTypeEnum,
+  ResultEnum,
+} from '../../../../../graphql/types.generated';
+import { ContactCheckBox } from '../../ContactCheckBox/ContactCheckBox';
+import { StarContactIcon } from '../../StarContactIcon/StarContactIcon';
+import { ContactTaskRowFragment } from './ContactTaskRow.generated';
+import { TaskCommentsButton } from './TaskCommentsButton/TaskCommentsButton';
+import { TaskCompleteButton } from './TaskCompleteButton/TaskCompleteButton';
+import { TaskDueDate } from './TaskDueDate/TaskDueDate';
+
+const TaskRowWrap = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  margin: theme.spacing(1),
+}));
+
+const TaskType = styled(Typography)(({ theme }) => ({
+  fontSize: 14,
+  fontWeight: 700,
+  color: theme.palette.text.primary,
+}));
+
+const TaskDescription = styled(Typography)(({ theme }) => ({
+  fontSize: 14,
+  color: theme.palette.text.primary,
+  marginLeft: theme.spacing(0.5),
+}));
+
+const Spacer = styled(Box)(({}) => ({ flex: 1 }));
```

Comment:
Idk if this is the best pattern. You can do stuff like this instead:
```html
<div style="display: flex; justify-content: space-between;">
  <div>Left stuff</div>
  <div>Right stuff</div>
</div>
```
(not suggesting inline styles, just writing quick code)

Or maybe CSS Grid could be better here since it is more like a table.